v1.1-beta/matrixx-digital-commerce/matrixx-tmf-party-management/cnf-testsuite.yml

```diff
@@ -0,0 +1,13 @@
+---
+helm_directory: "tmf-service-activation-management"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-232795837.yaml\
+  \ --devel"
```

Comment:
@deerowbear @daniel-wilmes if an end-user is replicating the testing will they use the --devel flag?

Reply:
Yes currently in our pipline that is what gets run.